features/bevragen/persoon/opschorting-bijhouding/dev/fields-gba.feature

```diff
@@ -0,0 +1,114 @@
+#language: nl
+
+Functionaliteit: GbaPersoon: opschorting bijhouding leveren als niet gevraagd met fields
+
+
+Rule: opschortingBijhouding wordt automatisch geleverd indien van toepassing
+
+  Abstract Scenario: 'reden opschorting bijhouding (67.20)' met waarde '<opschorting reden>'
+    Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende gegevens
+    | anummer (01.10) | voornamen (02.10) | voorvoegsel (02.30) | geslachtsnaam (02.40) | geboortedatum (03.10) | geslachtsaanduiding (04.10) |
+    | 0123456789      | William           | de                  | Vries                 | 20040526              | M                           |
+    En de persoon heeft de volgende 'inschrijving' gegevens
+    | datum opschorting bijhouding (67.10) | reden opschorting bijhouding (67.20) |
+    | 20020701                             | <opschorting reden>                  |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000152                       |
+    | fields              | naam,geboorte,geslacht          |
+    Dan heeft de response een persoon met de volgende gegevens
+    | naam                  | waarde |
+    | geslacht.code         | M      |
+    | geslacht.omschrijving | man    |
+    En heeft de persoon de volgende 'naam' gegevens
+    | naam          | waarde  |
+    | voornamen     | William |
+    | voorvoegsel   | de      |
+    | geslachtsnaam | Vries   |
+    En heeft de persoon de volgende 'geboorte' gegevens
+    | naam  | waarde   |
+    | datum | 20040526 |
+    En heeft de persoon de volgende 'opschortingBijhouding' gegevens
+    | naam               | waarde               |
+    | reden.code         | <opschorting reden>  |
+    | reden.omschrijving | <reden omschrijving> |
+    | datum.datum        | 20020701             |
+
+    Voorbeelden:
+    | opschorting reden | reden omschrijving        |
+    | O                 | overlijden                |
+    | E                 | emigratie                 |
+    | M                 | ministerieel besluit      |
+    | R                 | pl is aangelegd in de rni |
+    | .                 | onbekend                  |
+
+  Scenario: volledig onbekende datum Opschorting Bijhouding
+    Gegeven de persoon met burgerservicenummer '000000322' heeft de volgende 'inschrijving' gegevens
+    | naam                                 | waarde   |
```

Comment:
Kan reden leeg zijn? Of moet het de waarde . hebben?

Reply:
Als reden leeg is, is er geen opschorting. Reden leeg en datum gevuld (of andersom) is een data fout.